src/newLaunch/launchConfig.ts

```diff
@@ -36,15 +35,15 @@ export interface IProjectDetails {
   summary: string,
   proposition: string,
   teamDescription: string,
-  logo: string
+  logo?: string
```

Comment:
why is this marked optional?  should this actually be deleted?  I can't find where it is used.